Gifski/Gifski.swift

```diff
@@ -97,5 +96,7 @@ final class Gifski {
 			gifski_write(g, outputUrl.path)
 			gifski_drop(g)
 		}
+
+		return progress
```

Comment:
I wouldn't return the progress here. Is it because you might want to use KVO? If so I would use create a new progress, use progress.becomeCurrent.. and observe that instance instead..

Reply:
> Is it because you might want to use KVO?

Yes

> If so I would use create a new progress, use progress.becomeCurrent.. and observe that instance instead..

I agree. That's better. I just didn't fully realize how NSProgress works.